SofaKernel/modules/SofaObjectInteraction/src/SofaObjectInteraction/initSofaObjectInteraction.h

```diff
@@ -19,25 +19,10 @@
 *                                                                             *
 * Contact information: contact@sofa-framework.org                             *
 ******************************************************************************/
-#include <SofaObjectInteraction/initObjectInteraction.h>
+#pragma once
+#include <SofaObjectInteraction/config.h>
 
-
-namespace sofa
-{
-
-namespace component
+namespace sofa::component
```

Comment:
Why keeping this file at this point?

Reply:
at the beginning, just to get an header next to initBlaBla.cpp but yeah not useful, I concur

Reply:
for SofaBase modules, it is useful tho

Reply:
Yeah but for plugins, the initBlabla is only there to expose global functions to get informations about the plugin by the plugin manager, no need for a header file.

Reply:
Anyway, keep it or delete it, I don't really care, everything looks good to me, ready to be merged whenever you are ready 👍